client/platform/desktop/frontend/store/dataset.ts

```diff
@@ -43,7 +45,7 @@ const datasets = ref({} as Record<string, JsonMetaCache>);
 
 const recents = computed(() => {
   const list = Object.values(datasets.value)
-    .sort((a, b) => Date.parse(b.createdAt) - Date.parse(a.createdAt));
+    .sort((a, b) => Date.parse(b.accessedAt) - Date.parse(a.accessedAt));
```

Comment:
Now that you're doing sorting with v-data-table, can you just use a default sort on that table and remove this?

Or do you think it's better to do the default sort here?

Reply:
It's one of the things I need to check.  I may want to re implement the `sort` in the table to use the same `Date.parse`, because I imagine it's just sorting on the string name and I'm not sure if that is the best way.